### PR TITLE
Update vuejs.md

### PR DESCRIPTION
**Pull request by alOneh**

### CODE REVIEW

File: client-generator/vuejs.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Reference the Bootstrap CSS stylesheet in `index.html` (optional):
 
 In the app directory, generate the files for the resource you want:
 
-    $ generate-api-platform-client https://demo.api-platform.com src/ --resource foo
+    $ generate-api-platform-client https://demo.api-platform.com src/ --resource foo -g vue
```
Comment (dunglas) [inline client-generator/vuejs.md]: Can you use the long form?

**Comment by alOneh** [comment]
Ok, I did'nt see that in #293 it was already fixed 😅 , the online doc is not auto generated ?

**Comment by dunglas** [comment]
Not yet, we're working on it.

**Comment by alOneh** [comment]
We can close this as it was already done or you still want the long form ?

**Comment by dunglas** [comment]
I still but it's not very important :)

**Comment by dunglas** [comment]
Closing are there are a lot of merge conflicts

**Pull request closed without merging**